libraries/Microsoft.Bot.Builder.AI.QnA/Models/KnowledgeBaseAnswers.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// Contains answers for a user query.
+    /// </summary>
+    public class KnowledgeBaseAnswers
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="KnowledgeBaseAnswers"/> class. Initializes a new instance of answers.
+        /// </summary>
+        /// <param name="answers">The answers for a user query,
+        /// sorted in decreasing order of ranking score.</param>
+        public KnowledgeBaseAnswers(List<KnowledgeBaseAnswer> answers)
```

Comment:
We need to review how this is being used, we may be able to just initialize the Anserts property with a List<KnowledgeBaseAnswer> and remove this constructor